libs/ml/include/ml/core/node.hpp

```diff
@@ -153,258 +153,5 @@ class Node
   std::shared_ptr<ops::Ops<TensorType>> op_ptr_;
```

Comment:
There seems to be several methods you have not moved over to source file

Reply:
Maybe it is better to leave such one-line methods in header file and just make them inline?